processor/stash/stash.go

```diff
@@ -231,7 +231,7 @@ func (st *HandleT) storeErrorsToObjectStorage(jobs []*jobsdb.JobT) (errorJob []E
 			prefixes := []string{"rudder-proc-err-logs", time.Now().Format("01-02-2006")}
 			uploadOutput, err := errFileUploader.Upload(context.TODO(), outputFile, prefixes...)
 			errorJobs = append(errorJobs, ErrorJob{
-				jobs: jobsPerWorkspace[workspaceID],
+				jobs: jobsPerWorkspace[wrkId],
```

Comment:
please rename `wrkId` to `workspaceID` so that shadowing can protect us everywhere.
```
for workspaceID, filePath := range dumps {
		workspaceID := workspaceID
```

Reply:
Already merged this branch. Should I revert and make this change?

Reply:
we can do this change later in master branch.